lib/engine/game/g_18_mex/game.rb

```diff
@@ -69,7 +69,7 @@ class Game < Game::Base
           '470' =>
           {
             'count' => 1,
-            'color' => 'yellow',
+            'color' => 'orange',
```

Comment:
18mex?

Reply:
Yeah, 18MEX has a similar orange special tile, but it broke 18MEX so that can be added in a separate commit. Maybe.
Will add a commit that removes this change.